source/index.html.md

```diff
@@ -679,6 +678,57 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
+
+##### Method: `googlepay`
+
+To accept a payment using Google Pay, the complete payment token, recipient ID
+and derived shared secret, are required. Please refer to the [official
+documentation][GooglePay-PaymentCryptography]. Only protocol version `ECv2` is
+supported.
+
+<dl class="dl-vertical">
+  <dt>googlepay[token]
+    <span class="type">[:json:]</span>
+  </dt>
+  <dd>
+    Raw payment method token as received in response from Google. UTF-8 encoded
+    serialization of a JSON dictionary.
+  </dd>
+  <dt>googlepay[shared_key]
+    <span class="type">[:base64:]</span>
+  </dt>
+  <dd>
+    The shared secret derived from the ephemeral public key and your private
+    key.
+  </dd>
+  <dt>googlepay[recipient_id]
+    <span class="type">[\x20-\x7E]+
```

Comment:
Since space is not allowed:
```suggestion
    <span class="type">[\x21-\x7E]+
```

But also, isn't there an upper limit on the length?

I went through the documentation I have at hand and couldn't find any.